src/services/utilServices/RateLimiter.ts

```diff
@@ -0,0 +1,16 @@
+import rateLimit from "express-rate-limit"
+
+// NOTE: Refer here for more information regarding the implementation:
+// https://github.com/express-rate-limit/express-rate-limit
+// Also, note that our production environment has 2 instances
+// and the rate limiter uses an in memory store,
+// so our effective limit is 100 * 2.
+// This also implies that a client can hit the limit on 1 server
+// but not on the other, leading to inconsistent behaviour.
+// eslint-disable-next-line import/prefer-default-export
+export const rateLimiter = rateLimit({
+  windowMs: 15 * 60 * 1000, // 15 minutes
+  max: 100, // Limit each IP to 100 requests per `window` (here, per 15 minutes)
```

Comment:
We can sync up the window with the time window for otps, so that they reset at the same intervals.

Also can consider dropping the max  number of requests (maybe to 5? which should be more than enough) if we go with restricting only the actual verification

Reply:
i'm gonna change it to env var but holding off on `max` (see my point above)